assets/setup-wizard/purpose/purpose-item.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { CheckboxControl } from '@wordpress/components';
+
+/**
+ * Purpose Item component.
+ *
+ * @param {Object}   props          Component props.
+ * @param {string}   props.label    Item label.
+ * @param {boolean}  props.checked  Whether it's checked.
+ * @param {Function} props.onToggle Toggle callback.
+ * @param {Object}   props.children Component children, which is displayed when it's checked.
+ */
+const PurposeItem = ( { label, checked, onToggle, children } ) => (
+	<li className="sensei-setup-wizard__purpose-item">
```

Comment:
In the design, the border seems to be a light gray on unchecked items. Would that be difficult to implement? It might be a nice visual cue.

Currently:
<img width="554" alt="Screen Shot 2022-09-23 at 6 07 42 PM" src="https://user-images.githubusercontent.com/68693/192015767-6f4decec-2c04-4422-8cc7-226da7de16d3.png">

Design: 
<img width="262" alt="Screen Shot 2022-09-23 at 6 11 03 PM" src="https://user-images.githubusercontent.com/68693/192016047-f4fb3911-7375-4fdf-9a68-0891b672c881.png">

Reply:
Good catch! Fixed here: https://github.com/Automattic/sensei/pull/5733/commits/56085b452e23f2d8edf4c071a3f9b10701a05c6a

Reply:
Sorry! I was talking about the border around the checkbox and label, not the checkbox.

Reply:
Oops! Also fixed here: https://github.com/Automattic/sensei/pull/5733/commits/341ccb3e754a95b603b6de9ba731f7c0d9da0919